### DISCUSSION TITLE
adding fontawesome to admin and gutenberg block editor

### DISCUSSION THREAD

**Discussion by yoscbd** [holger1411/understrap#1902] [Q&A] [answered]
Hi there,
Wanted to know how should i include support for fontawesome in my backend (Gutenberg block editor).

At the moment it is only avialble on the front end.

Thank you all!

**Comment by bacoords** [comment, marked as answer]
You should just be able to add the import to the `src/sass/custom-editor-styles.scss` file:

`@import "assets/fontawesome/font-awesome";                           // <------- Font Awesome Icon font`

I believe that will handle pulling in the font files themselves, but I haven't tested this.
